modules/configuring-firewall.adoc

```diff
@@ -48,27 +48,18 @@ Before you install {product-title}, you must configure your firewall to grant ac
 |===
 |Cloud |URL |Function
 
-.2+|AWS
-
-|`route53.amazonaws.com`
-|Allows the Cluster Ingress Operator to creates wildcard alias records for routes
-
-|`amazonaws.com`
-|Required to access AWS services and resources, including s3, iam, and `quay-registry.s3`
+|AWS
+|`*.amazonaws.com`
+|Required to access AWS services and resources, including s3, iam, and `quay-registry.s3`. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
 
 |GCP
-|`www.googleapis.com/dns/v1/projects`
-|cluster-ingress-operator: Creates wildcard alias records for routes.
-
-.3+|Azure
-|`management.azure.com/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/dnsZones`, where `<subscriptionId>` and `<resourceGroupName>` are the subscription ID and resource group name that you use for your cluster
-|Allows the Cluster Ingress Operator to creates wildcard alias records for routes
+|`*.googleapis.com`
+|Allows the Cluster Ingress Operator to creates wildcard alias records for routes. Review link:https://cloud.google.com/endpoints/[Cloud Endpoints] in the GCP documentation to determine the endpoints to allow for your APIs.
```

Comment:
Again, lose "Allows the Cluster Ingress Operator to creates wildcard alias records for routes." We use these endpoints for all parts of the installation and talk to many services for lots of different reasons (not just ingress).